orchestra/models.py

```diff
@@ -461,6 +461,43 @@ def __str__(self):
             str(self.task), self.assignment_counter, str(self.worker))
 
 
+class Iteration(models.Model):
+    """
+    Iterations are the contiguous units of a worker's time on task.
+
+    Attributes:
+        start_datetime (datetime.datetime):
+            The time the task was assigned for this iteration.
+        end_datetime (datetime.datetime):
+            The time the iteration was completed. Will be None if the iteration
+            is currently in progress.
+        assignment (orchestra.models.TaskAssigment):
+            The task assignment to which the iteration belongs.
+        action (orchestra.models.Iteration.Action):
+            Represents the action that was taken at the end of the iteration.
+            Will be None if the iteration is currently in progress.
+        submitted_data (str):
+            A JSON blob containing submitted data for this iteration. Will be
+            None if the iteration is currently in progress.
+    """
+
+    class Action:
+        SUBMIT = 0
+        ACCEPT = 1
+        REJECT = 2
+
+    ACTION_CHOICES = (
+        (Action.SUBMIT, 'Submit'),
+        (Action.ACCEPT, 'Accept'),
+        (Action.REJECT, 'Reject'))
+
+    start_datetime = models.DateTimeField(default=timezone.now)
```

Comment:
Right now projects, tasks, assignments, workers all have fields called `start_datetime`—probably easier to be consistent for now and change them all in one go if we decide to?

(I know I advocated for inconsistency above, don't hate me too much for my hypocrisy :yum:)